src/app/dim-ui/CustomStatTotal.tsx

```diff
@@ -47,16 +48,16 @@ export function StatTotalToggle({
         ].map(({ className, includesCheck }) => (
           <span key={className} className={clsx(className, { [styles.readOnly]: readOnly })}>
             {addDividers(
-              armorStats
-                .filter((statHash) => activeStats.includes(statHash) === includesCheck)
-                .map((statHash) => (
+              filterMap(armorStats, (statHash) =>
+                activeStats.includes(statHash) === includesCheck ? (
```

Comment:
This is perhaps the one case where I don't think it's actually an improvement for readability, since the filter predicate and the mapping function are completely independent of each other. In most other cases the filter depends on the mapping function (e.g. `compact`) or the mapping function depends on the filter (e.g. check for `plugged` in filter, then type assertion for `plugged!` in map), but I don't know about this one.

Reply:
Agreed - I did a couple of these and decided the same, I was going to go through and revert them.